privacy-dashboard/privacy-dashboard-impl/src/main/java/com/duckduckgo/privacy/dashboard/impl/ui/PrivacyDashboardHybridViewModel.kt

```diff
@@ -186,6 +223,22 @@ class PrivacyDashboardHybridViewModel @Inject constructor(
         viewModelScope.launch { updateSite(site) }
     }
 
+    private fun pixelParamList(): List<String> {
+        // we're only running for english speaking users initially.
+        val isEnglish: Boolean = Locale.getDefault().language == Locale.ENGLISH.language
+        if (!isEnglish) return emptyList()
+
+        // if protectionToggleHighlightActive is not enabled, send no params
+        if (!privacyDashboardRemoteFeature.protectionToggleHighlight().isEnabled()) return emptyList()
+
+        // otherwise, send the pixel param
+        return listOf(PixelParameter.PROTECTION_TOGGLE_EXP)
+    }
+
+    private fun pixelParamMap(): Map<String, String> {
+        return pixelParamList().associateWith { true.toString() }
+    }
+
     private suspend fun updateSite(site: Site) {
         withContext(dispatcher.main()) {
```

Comment:
This change goes alongside the one above in `RemoteFeatureSettingsViewState` and the one below this one in the same method (GH is stupid enough to not allow suggestions in code that isn't modified, so I needed to split it)
 ```suggestion
        suspend fun createRemoteFeatureSettingsViewState() = withContext(dispatcher.io()) {
            val layout = if (privacyDashboardRemoteFeature.protectionToggleHighlight().isEnabled()) {
                LayoutType.HIGHLIGHTED_PROTECTIONS_TOGGLE.value
            } else {
                LayoutType.DEFAULT.value
            }

            return@withContext RemoteFeatureSettingsViewState(
                primaryScreen = PrimaryScreenSettings(
                    layout = layout,
                ),
            )
        }
        val remoteFeatureSettings = createRemoteFeatureSettingsViewState()
        withContext(dispatcher.main()) {
```